port_ocean/config/settings.py

```diff
@@ -78,6 +78,7 @@ class IntegrationConfiguration(BaseOceanSettings, extra=Extra.allow):
         default_factory=lambda: IntegrationSettings(type="", identifier="")
     )
     runtime: Runtime = Runtime.OnPrem
+    resources_path: str = Field(default=".port/resources")
```

Comment:
pydantic has property type of path 
https://docs.pydantic.dev/1.10/usage/types/

```
Pydantic Types[¶](https://docs.pydantic.dev/1.10/usage/types/#pydantic-types)
pydantic also provides a variety of other useful types:

FilePath
like Path, but the path must exist and be a file
DirectoryPath
like Path, but the path must exist and be a directory
```

Reply:
Hmmmm this is interesting, but in case of a wrong directory it will explode instead of just using the backoff one, no?
Also I don't really like this error message 😅 
![image](https://github.com/user-attachments/assets/2f72974b-f335-4b62-a529-a4cf29cc18f1)